<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadExample</class>
 <widget class="QWidget" name="ThreadExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thread Example</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="toolTip">
        <string>Creats a new scenaro than creates a new platform and uses a thread to generate the position data</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="toolTip">
        <string>Stops the thread and puts the scenario in File mode</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The number of data points added to the platform</string>
       </property>
       <property name="text">
        <string>Number of Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
